pytezos-tests/README.md

```diff
@@ -19,7 +19,7 @@ Set up python3 virtual environment.
 We'll need to download [PyTezos](https://github.com/baking-bad/pytezos) to interact with the tezos node. 
 ```
     sudo apt-get install libsodium-dev libsecp256k1-dev libgmp-dev
-    pip3 install pytezos
+    pip3 install requirements
```

Comment:
btw, i think you meant: `pip3 install -r requirements.txt`

Reply:
fixed